docs/style_guide.md

```diff
@@ -0,0 +1,16 @@
+# Style guide
+
+## Imports (`use` keyword)
+
+Each symbol should be on a separate line (to simplify merging).
```

Comment:
I think we don’t need to write things CI enforces here, right? A machine can tell people when it’s wrong and a machine can fix it, so there’s no need for human nitpicking or toil.